app/views/thread_header.html

```diff
@@ -6,11 +6,19 @@ <h3 class="question-cat">
   , {% endif %}
   {{ displayable_metadata.operator }}
 </h3>
-<select id="category_chooser" class="button-gray">
-  {% for key, value in categories %}
-    <option value="{{ key }}"
-      {% if selected_category == key %}selected{% endif %}>
-      {{ value }}
-    </option>
-  {% endfor %}
-</select>
+
+{# This <ul><li> is needed to get the right Building block styling for that button #}
+<ul>
+  <li>
+    <div class="button" data-icon="forward">
```

Comment:
The size is set by the building blocks to 1.6rem, which is 16px for us so it should be the proper size per building block styling.
Also, the examples on http://buildingfirefoxos.com/building-blocks/buttons.html use forward and not forward-light. So I'll just fix the italic issue for now.